Mathlib/Topology/Instances/ENNReal.lean

```diff
@@ -1117,6 +1117,28 @@ theorem summable_of_le {f g : β → ℝ≥0} (hgf : ∀ b, g b ≤ f b) : Summa
     hp.summable
 #align nnreal.summable_of_le NNReal.summable_of_le
 
+/-- Summable non-negative functions have countable support -/
+theorem _root_.Summable.countable_support (f : α → ℝ≥0) (h : Summable f) :
```

Comment:
```suggestion
theorem _root_.Summable.countable_support_nnreal (f : α → ℝ≥0) (h : Summable f) :
```
then you could provide a short wrapper for `real` that takes the hypothesis that `f a` is nonnegative everywhere.

Reply:
Sure thing!